frontend/packages/console-app/src/actions/creators/pvc-factory.ts

```diff
@@ -0,0 +1,57 @@
+import i18next from 'i18next';
+import {
+  clonePVCModal,
+  expandPVCModal,
+  restorePVCModal,
+} from '@console/internal/components/modals';
+import { asAccessReview } from '@console/internal/components/utils';
+import { VolumeSnapshotModel } from '@console/internal/models';
+import { VolumeSnapshotKind } from '@console/internal/module/k8s';
+import { ResourceActionFactory } from './common-factory';
+
+export const PVCActionFactory: ResourceActionFactory = {
```

Comment:
I cannot find a usage for this... is this used?

Reply:
No, currently its not being used. I just converted all the factory actions from the old type to new type so that it would be easier to migrate other resource actions easily later.